kubeadm/golang/windowsservercore-1909/Dockerfile

```diff
@@ -0,0 +1,73 @@
+FROM mcr.microsoft.com/windows/servercore:1909

```

Comment:
This is fine for now but it'd be cool if we could upstream this to https://github.com/docker-library/golang

Reply:
Ok i will submit a copy to